api/floor/schema.py

```diff
@@ -78,19 +80,22 @@ def mutate(self, info, floor_id, **kwargs):
 class DeleteFloor(graphene.Mutation):
 
     class Arguments:
+        status = graphene.String()
         floor_id = graphene.Int(required=True)
     floor = graphene.Field(Floor)
 
     @Auth.user_roles('Admin')
     def mutate(self, info, floor_id, **kwargs):
         query_floor = Floor.get_query(info)
-        exact_floor = query_floor.filter(
+        resultActive = query_floor.filter(FloorModel.status == "active")
```

Comment:
You did not follow the naming convention for this variable and also the name could be more descriptive.

Reply:
@omobosteven , I have implemented the changes